### ISSUE TITLE
errorCode=1 byte-range-spec must be positive

### ISSUE THREAD

**Issue by sky93** [tatsuhiro-t/aria2#509]
Hello.

When I download a link like this:
`aria2c "http://tb27.trainbit.com:8080/files/6135087884/v3664B3059785465766E6956736877307A45416C68372B436636594F6E67487445615441447930764B2B54673D/"`

aria2 says: `errorCode=1 byte-range-spec must be positive`

I can download the file using `wget` or ... but it's impossible using aria2. What is wrong?

**Comment by tatsuhiro-t** [comment]
Could you paste the response headers from the web site here?

**Comment by sky93** [comment]
I used curl to get response header: 
```
$ curl -i "http://tb27.trainbit.com:8080/files/6135087884/v3664B3059785465766E6956736877307A45416C68372B436636594F6E67487445615441447930764B2B54673D
/"


HTTP/1.1 200 OK
Content-Type: application/octet-stream
Connection: keep-alive
Accept-Ranges: bytes
Content-Disposition: attachment; filename=The_Game_Awards_2015.part1.rar
Content-Le
![error](https://cloud.githubusercontent.com/assets/8404511/11912542/2cbb3fa4-a656-11e5-8e42-31613bacc893.jpg)
ngth: 2096103424
Content-Range: bytes=0-2096103423/2096103424
```

**Comment by tatsuhiro-t** [comment]
The cause is this:
```
Content-Range: bytes=0-2096103423/2096103424
```

This is violation of RFC 7233.  This kind of representation is for client request, not response from server.  I think this server implementation misunderstood this.

Anyway, HTTP/1.1 is full of legacy implementation, we will do some hack to support this...

**Comment by tatsuhiro-t** [comment]
Fixed in 8512fe9

**Comment by sky93** [comment]
Thanks for the fix :+1:
